Source/WebCore/workers/shared/context/SharedWorkerContextManager.h

```diff
@@ -55,6 +55,7 @@ class SharedWorkerContextManager {
         virtual void postErrorToWorkerObject(SharedWorkerIdentifier, const String& errorMessage, int lineNumber, int columnNumber, const String& sourceURL, bool isErrrorEvent) = 0;
         virtual void sharedWorkerTerminated(SharedWorkerIdentifier) = 0;
         bool isClosed() const { return m_isClosed; }
+        virtual bool isContextManagerConnection() const { return true; }
```

Comment:
This fella wants to return false too.

Possibly also rename this to `isWebSharedWorkerContextManagerConnection`.